package/Shaders/Lighting.hlsl

```diff
@@ -1503,18 +1500,30 @@ PS_OUTPUT main(PS_INPUT input, bool frontFace
 	float maxOcclusion = 1;
 	float minWetnessAngle = 0;
 	minWetnessAngle = saturate(max(minWetnessValue, worldSpaceNormal.z));
+
 #		if defined(SKYLIGHTING)
-	float skylight = GetSkylightOcclusion(input.WorldPosition + worldSpaceNormal, screenNoise);
-#		else
-	float skylight = 1.0;
-#		endif  // SKYLIGHTING
+#			if defined(DEFERRED)
+	float wetnessOcclusion = WetnessOcclusionTexture.Load(int3(screenUV * BufferDim, 0));
```

Comment:
This doesn't seem to work in VR, does it work in flatrim?

Reply:
Is that texture stereo? If so, the uv would have to be converted.

Reply:
Not sure, tried to convert it. Still didn't get it to work so not sure. Maybe need to update several places